README.md

```diff
@@ -21,8 +21,8 @@ Locally importing is used typically while testing new SDK method implementations
 First build the package `npm run build`
 
 Make sure `src/enums.ts` exists. If it does not, run `npm run transpile` **(must be logged in to Github package registry)**
-```typescript
-import CE, { Enums } from 'path/to/ts-sdk/root';
+```
+import CE, { Enums } from path/to/ts-sdk/root';
```

Comment:
```suggestion
import CE, { Enums } from 'path/to/ts-sdk/root';
```

Reply:
I recently learned there's a cleaner way to write this,

```
import { default as CE, Enums } from 'path/to/ts-sdk/root';
```
I think it looks nicer this way and more obvious that CE = default export but up to you if you want to change it.

https://stackoverflow.com/a/39282290/2805355